src/lib/relay/routes.tsx

```diff
@@ -18,6 +18,24 @@ class Artist extends Relay.Route {
   static routeName = "ArtistRoute"
 }
 
+class Conversation extends Relay.Route {
+  static queries = {
+    me: (component, params) => Relay.QL`
```

Comment:
I do not, but also haven’t tried. I’d just add the extra route for now, though, maybe with Relay Modern this gets simplified anyways.